common.mk

```diff
@@ -135,7 +136,7 @@ busybox-cleaner-common:
 LINUX_COMMON_FLAGS ?= LOCALVERSION= CROSS_COMPILE=$(CROSS_COMPILE_NS_KERNEL)
 
 linux-common: linux-defconfig
-	$(MAKE) -C $(LINUX_PATH) $(LINUX_COMMON_FLAGS)
+	$(MAKE) -C $(LINUX_PATH) -j$(NPROC) $(LINUX_COMMON_FLAGS)
```

Comment:
This will not play nicely with `make -jN`. It will generate a warning (`make[1]: warning: -jN forced in submake: disabling jobserver mode`) and you will likely end up running more jobs than requested. For example: `make -j1` will still spawn `$(NPROC)` jobs here.

We would need to do `MAKEFLAGS += -j$(shell nproc)` globally, but only if `$(MAKEFLAGS)` does not already contain `-j`, which I can't find a way to detect unfortunately :( at least with GNU make 4.0.